docs/usage/event-notifications.md

```diff
@@ -0,0 +1,72 @@
+---
+id: event-notifications
+title: "Event notifications"
+sidebar_position: 1
+---
+
+# Event notifications
+
+Botkube sends notifications according to the [sources](../configuration/source.md) configuration.
+
+## Actionable notifications
+
+If you have [`kubectl` executor enabled](../configuration/executor.md) for a given channel, you can run commands related to a resource from the notification itself. Use the dropdown on the left to select and run a given command:
+
+![Actionable notifications](./assets/actionable-notifications.png)
+
+The command dropdown is disabled for resource deletion events. It uses executor bindings to determine which commands are available for a given resource.
+
+:::info
+Actionable notifications are only available for the [Slack integration](../installation/socketslack/index.md) that supports interactive messages. Currently, only a selected list of commands are supported, such as `describe`, `get`, or `logs`.
```

Comment:
Just a thought, maybe we can introduce simple labels for each integration and use it similar to Slack:
![Screen Shot 2022-10-17 at 14 50 17](https://user-images.githubusercontent.com/17568639/196181454-d19c24f0-51f8-4dd1-bb05-95b0bec2256e.png)

For us it can be, e.g.: "Supported platforms: [Slack] [Discord]"

Reply:
Good idea, but I'd postpone that as we'd need to:
- have a custom component for that and use MDX (easy)
- review all the docs we have and put proper labelling everywhere

In the meantime, can I put this feature to the feature matrix? https://botkube.io/docs/installation/#feature-map